Documentation/CodingGuidelines/CglPhp/PhpArchitecture/WorkingWithExceptions.rst

```diff
@@ -67,15 +67,15 @@ Typical cases for exceptions that are designed to be caught
 Typical cases for exceptions that should not be caught
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* Wrong configuration: A flex from contains a :code:`type=inline` field.
+* Wrong configuration: A flex form contains a :code:`type=inline` field.
```

Comment:
I do believe flex form should be spelled flexform. See for example https://docs.typo3.org/typo3cms/extensions/sitepackage/CustomContentElements/TheSliderElement/Flexform/Index.html

I will fix this and merge.